reference-architectures/deploy-arch-ibm-slz-vpc-standard.md

```diff
@@ -0,0 +1,125 @@
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+compliance: FedRAMP
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+<!--
+Don't include "reference architecture" in the following title.
+Specify a title based on a use case. If the architecture has a module
+or tile in the IBM Cloud catalog, match the title to the catalog. See
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# VPC landing zone - Standard variation
+{: #vsi-ra-qs}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+
+The Standard variation of the VPC landing zone deployable architecture uses two Virtual Private Clouds (VPC), a Management VPC, and a Workload VPC to manage the environment and the deployed workload. Each VPC is a multi-zoned, multi-subnet implementation that keeps your workloads secure.
+
+Virtual Private Endpoints connect the VPCs to each other and to IBM Cloud services. Cloud services include Key Protect, IAM, Container Registry, and Cloud Object Storage. The IBM Cloud services are configured for storage and key management.
+
+IBM Cloud Flow Logs for VPC enables the collection, storage, and presentation of information about the internet protocol (IP) traffic that is going to and from network interfaces within your VPC. In addition, Activity Tracker logs events from enabled services. IBM Cloud Flow Logs for VPC and Activity Tracker are included in this deployable architecture. You can add more security services, such as Hyper Connect Crypto Services.
+
+
+## Architecture diagram
+{: #ra-vpc-architecture-diagram}
+
+![Architecture diagram for the Standard variation of VPC landing zone](vpc.drawio.svg "Architecture diagram of VPC landing zone deployable architecture"){: caption="Figure 1. Standard variation of VPC landing zone" caption-side="bottom"}
+
+## Design requirements
+{: #ra-vpc-qs-design-requirements}
+
+![Design requirements for VPC landing zone](heat-map-deploy-arch-slz-vpc-standard.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
+
+<!--
+TODO: Add the typical use case for the architecture.
+The use case might include the motivation for the architecture composition,
+business challenge, or target cloud environments.
+-->
+## Components
+{: #ra-vpc-components}
+
+### VPC architecture decisions
+{: #ra-vpc-components-arch}
+
+| Requirement | Component | Reasons for choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+| * Provide infrastructure administration access  \n * Limit the number of infrastructure administration entry points to ensure security audit | Management VPC service | Create a separate VPC service where SSH connectivity from outside is allowed | |
```

Comment:
Requirement for the VPC (Management/Workload) should be consistent with the vsi-quickstart pattern for all of them

I am not sure what network access is there by default since it looks like some updates.  @ocofaigh / @vburckhardt do you know?

Reply:
Updated to this to match the VSI QS requirement for the Management VPC service:

* Provide infrastructure/application administration access to monitor, operate, and maintain the environment.
* Limit the number of infrastructure/application administration entry points to ensure security audit.

Reply:
Updated in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/41